translate/convert/test_po2ts.py

```diff
@@ -77,8 +77,7 @@ def test_duplicates(self):
 msgstr "b"
 '''
         tsfile = self.po2ts(posource)
-        print(tsfile)
```

Comment:
I think they are for debugging, for when some test fails and you can simply go up and see what the prints did output before the failure. Does this provide some noticeable performance improvement? If yes please move this to a separate PR.